connexion/apis/aiohttp_api.py

```diff
@@ -27,7 +29,7 @@
 
 @web.middleware
 @asyncio.coroutine
-def oauth_problem_middleware(request, handler):
+def generic_problem_middleware(request, handler):
```

Comment:
This will need to be rebased on master as #952 was just merged.
Now, aiohttp turns all exceptions into problems by default.

This will need to at least replace `exc.to_problem()` with `problem_from_object(exc)`.

Reply:
@badcure when you rebase, please also update _generic_problem which has been added to this file.